.github/workflows/test.yml

```diff
@@ -46,7 +47,7 @@ jobs:
           pip install -r ./requirements.txt
 
       - name: Run tests
-        run: pytest --mypy-ini-file=mypy.ini
+        run: pytest --mypy-ini-file="${{ matrix.mypy_ini_file }}"
```

Comment:
Hm, good idea. Let's try this. You can create a new PR with just this change. It will be helpful!

Reply:
Yeah, I removed it as it's no longer needed to reproduce the problem after adding `disable_cache: true` to the test case. If we're going to add this I think it makes sense to rename the files as well, but I stole this from #1017 so I think I'll leave it up to the author there to decide if he wants to create a PR for that.